apache/superset
# feat(nginx): add static resource configuration

### SUMMARY

This PR adds static resource configuration to the Nginx setup in the Docker directory. The changes include specifying a static directory in nginx.conf to serve frontend assets with caching enabled. This update helps improve resource loading times and serves assets directly from the designated directory, making the Nginx configuration more aligned with application needs.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS

To manually verify these changes:

	1.	Stop the current docker-compose service.
	2.	Modify the nginx.conf file as specified:
	•	Add a /static alias pointing to the frontend source directory with caching enabled.
	3.	Restart docker-compose:
	•	docker compose up
        4. Check that static resources are served correctly under /static.

### ADDITIONAL INFORMATION
